<launch>
  <!-- <arg name="method" value="m2dp" /> -->

  <!-- <arg name="date" value="2014-07-14-14-49-50" /> -->
  <!-- <arg name="date" value="2014-11-28-12-07-13" /> -->
  <!-- <arg name="date" value="2014-12-12-10-45-15" /> -->
  <!-- <arg name="date" value="2015-02-10-11-58-05" /> -->
  <!-- <arg name="date" value="2015-05-19-14-06-38" /> -->
  <!-- <arg name="date" value="2015-05-22-11-14-30" /> -->
  <!-- <arg name="date" value="2015-08-13-16-02-58" /> -->
  <!-- <arg name="date" value="2015-10-30-13-52-14" /> -->
  
  <arg name="folder" value="$(find so_dso_place_recognition)/place_recognition/results/RobotCar/$(arg date)" /> 
  <node name="test_$(arg method)" pkg="so_dso_place_recognition" type="test_$(arg method)" output="screen">
    <param name="poses_history_file" value="$(arg folder)/poses_history_file.txt" />
    <param name="pts_history_file" value="$(arg folder)/pts_history_file.txt" />
    <param name="$(arg method)_file" value="$(arg folder)/history_$(arg method).txt" />
    <param name="incoming_id_file" value="$(arg folder)/incoming_id_file.txt" />
    <param name="lidarRange" value="45.0" />
  </node>
</launch>
